keyboards/dztech/dz60rgb_ansi/readme.md

```diff
@@ -5,6 +5,8 @@
 A hotswap 60% RGB keyboard.
 
 * Keyboard Maintainer: [moyi4681](https://github.com/moyi4681)
+* Vial Support: [dc959](https://github.com/dc959)
+    > Vial support currently only tested and working on DZ60RGB_ansi v2
```

Comment:
You shouldn't modify the original readme. If the keymap only works for a specific revision of the keyboard, you should put this keymap under that revision i.e. under `keyboards/dztech/dz60rgb_ansi/v2/keymaps`

Reply:
I did have a feeling that'd be the case.

I have moved the vial keymap under a keymaps folder in v2 and have reverted the readme commit